packages/fraction-model/src/main.jsx

```diff
@@ -121,15 +122,19 @@ export class Main extends React.Component {
   render() {
     const { model, classes } = this.props;
     const { showCorrect, session, answerChangeDialog } = this.state;
-    const { correctness = {}, language } = model;
+    const { prompt, title, correctness = {}, language } = model;
     const showCorrectAnswerToggle = correctness.correctness && correctness.correctness !== 'correct';
     const fractionModelChartKey = this.generateRandomKey();
 
     return (
       <div>
         <div className={classes.modelPreview}>
-          <h3 className={classes.titleContainer} dangerouslySetInnerHTML={{ __html: model.title }}></h3>
-          <p dangerouslySetInnerHTML={{ __html: model.prompt }}></p>
+          <h3 className={classes.titleContainer}>
+            <PreviewPrompt className="prompt" prompt={title} />
+          </h3>
+          <p>
+            <PreviewPrompt className="prompt" prompt={prompt} />
```

Comment:
you don't need to wrap `PreviewPrompt` in a `p` tag. You can add pass the tag you want for the text to the `PreviewPrompt` component (by default it's a `div`):
`<PreviewPrompt className="prompt" prompt={prompt} tagName="p" />`
same thing for the title

Reply:
made changes as per suggestion